packages/driver/cypress/fixtures/multi-domain-secondary.html

```diff
@@ -16,6 +17,10 @@
       document.querySelector('[data-cy="cypress-check"]').innerText = 'Has window.Cypress'
     }
 
+    if (window.testSecondaryDomainGlobal){
```

Comment:
adding a script to execute if the `window:before:load` event is fired.

Reply:
Instead of having this global, you could call `done()` in the test in the `cy.on('window.before.load')` event. If not, I would rename this to something like `testSecondaryWindowBeforeLoad`.

Reply:
I can rename the global. The intent of this script is to ensure that `window:before:load` gets called before this script as provided by the html is executed. By switching this over to call done(), we'd ensure the `window:before:load` event is called, but not when it is called. (unless I'm missing something).